static/bidder-info/silverpush.yaml

```diff
@@ -0,0 +1,13 @@
+endpoint: "https://prebid.chocolateplatform.co/bidder/?identifier=prebidchoc" 
+maintainer:
+  email: "prebid@silverpush.co"
+gvlVendorID: 42
```

Comment:
This doesn't seem like the correct GVL ID. Please confirm.

Reply:
Fixed
[commit](https://github.com/prebid/prebid-server/pull/2957/commits/cc30407f798a60b4550738f3802ffe7a19f81493)